clients/csharp-uwp/UWPVoiceAssistantSample/DirectLineSpeechDialogBackend.cs

```diff
@@ -242,7 +265,7 @@ public async Task StartAudioTurnAsync(bool performConfirmation)
         public async Task CancelSignalVerification()
 #pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously

```

Comment:
This pragma can go away now. Several good ways of doing it in general:
1. Make it not-async if it doesn't need to be async (sometimes interface definitions restrict you on being able to do this)
2. Await something if there's something to await (this change does that)
3. Remove the `async` and return Task.FromResult(0) at the end (`async` isn't part of the signature, just the method impl strategy, so this is legit).